src/main/java/com/google/devtools/build/lib/analysis/RuleContext.java

```diff
@@ -409,6 +410,34 @@ public ActionOwner getActionOwner() {
     return getActionOwner(DEFAULT_EXEC_GROUP_NAME);
   }
 
+  /**
+   * Returns a special action owner for test actions. Test actions should run on the target platform
+   * rather than the host platform. Note that the value is not cached (on the assumption that this
+   * method is only called once).
+   */
+  public ActionOwner getTestActionOwner() {
+    PlatformInfo executionPlatform;
```

Comment:
Name these `testExecutionPlatform` and `testExecProperties`, so clarify the distinction between them and the 'normal" exec platform.

Reply:
Good call, done!